atom/browser/resources/win/atom.manifest

```diff
@@ -32,7 +32,6 @@
 
   <asmv3:application xmlns:asmv3="urn:schemas-microsoft-com:asm.v3">
     <asmv3:windowsSettings xmlns="http://schemas.microsoft.com/SMI/2005/WindowsSettings">
-      <dpiAware>true</dpiAware>
```

Comment:
you can use `<dpiAware>True/PM</dpiAware>` instead of adding back the code
http://stackoverflow.com/questions/23551112/how-can-i-set-the-dpiaware-property-in-a-windows-application-manifest-to-per-mo